src/views/MobileDownloadLinks.tsx

```diff
@@ -0,0 +1,107 @@
+import styled from 'styled-components';
+
+import { ButtonShape, ButtonType } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import { IconName } from '@/components/Icon';
+import { IconButton } from '@/components/IconButton';
+import { Link } from '@/components/Link';
+import { VerticalSeparator } from '@/components/Separator';
+
+export const MobileDownloadLinks = ({ withBadges }: { withBadges?: boolean }) => {
+  const stringGetter = useStringGetter();
+
+  const androidUrl = document
+    .querySelector('meta[name="smartbanner:button-url-google"]')
```

Comment:
I don't love running these selectors on every render. Could we just throw them in a useMemo to run on mount?

Reply:
consolidated!